.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: required
+
+language: python
+
+python:
+  - "3.5"
+
+services:
+  - docker
+  
+before_script:
+  - docker-compose -f docker-compose.yml -f docker-compose-ci.yml build
```

Comment:
Change these lines on this one:
`docker-compose -f docker-compose.yml -f docker-compose-ci.yml up --build -d `
It should resolve a problem with setting an env variable to a container.